public/js/auth.js

```diff
@@ -55,9 +56,41 @@ if (formSignup) {
         }
         displayAlert(res.message, 2);
       })
-        .catch((err) => { displayAlert(err.message, 1); });
+      .catch((err) => { displayAlert(err.message, 1); });
     } else {
       displayAlert('Please fill in the form', 2);
     }
   });
 }
+
+if (formLogin) {
+  formLogin.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const email = document.getElementById('email').value;
+    const password = document.getElementById('password').value;
+
+    if (email && password) {
+      const body = { email, password };
+      fetch(`${baseUrl}/auth/login`, {
+        method: 'POST',
+        body: JSON.stringify(body),
+        headers: { 'Content-Type': 'application/json' },
+        mode: 'cors',
```

Comment:
why is course being enabled here ?

Reply:
I used it to test for Internet Explorer.
But forgot to remove it